Sem. 06/Pract.06/Task10.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+constexpr int MAX_SIZE = 50;
+
+void gettingTheNumbersIn(int arr[], int n) {
+    for (int i = 0; i < n; i++) {
+        std::cin >> arr[i];
+    }
+}
+
+bool isItInTheArray(int arr[], int n) {
+    bool numberIsInArray[n + 1] = {false};
```

Comment:
This array must be with constant size but it's not. 
You can initialize it this way: `bool numberIsInArray[MAX_SIZE] = {false};`
Also, it is better to set each array member to `false` explicitly because the initialization you are using is compilator-specific.